### ISSUE TITLE
Will you keep updating?

### ISSUE THREAD

**Issue by FrosenMercure** [MMD-Blender/blender_mmd_tools_append#108]
I see there are some issues with your addons and blender version 4.0 or higher, will you keep updating these addons?

**Comment by UuuNyaa** [comment]
Yes. I will continue to update this addon for the latest Blender LTS.

You can check the policy for the Blender versions at the following URL:
https://mmd-blender.fandom.com/wiki/MMD_UuuNyaa_Tools#Requirements

**Comment by Elypha** [comment]
@UuuNyaa hi would you please update the github wiki (https://github.com/UuuNyaa/blender_mmd_uuunyaa_tools/wiki#requirements) as well for the supported blender versions?
pasting the fandom wiki link there should just work since people may just not know the other wiki on fandom and could get outdated information
